Header header

float64 vel
float64 acc
---
bool success